superset/templates/odh_cr.yaml

```diff
@@ -104,7 +104,21 @@ spec:
       AUTH_ROLE_ADMIN = 'Admin'
 
       PUBLIC_ROLE_LIKE = 'Gamma'
-      
+      AUTH_ROLES_MAPPING = {
+      'cn=mhicks-all,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['CCX'],
+      'cn=candlepin,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['Subscriptions'],
+      'cn=cee-pel,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['Subscriptions']
+      }
+
+      # the LDAP user attribute which has their role DNs AUTH_LDAP_GROUP_FIELD =
+      AUTH_LDAP_GROUP_FIELD= 'memberOf'
+
+      # if we should replace ALL the user's roles each login, or only on
+      registration AUTH_ROLES_SYNC_AT_LOGIN = False
```

Comment:
This doesn't seem to work the way it is. Did you test if this line will work?

Reply:
I think we want this value to be true.

Reply:
ah yup, I forgot to change this from my dev environment 🤕

Reply:
I meant the `registration` word at the beginning of this line. It seems like it's part of the code, not the comment.